pom.xml

```diff
@@ -284,7 +284,11 @@
                     <!-- CircleCI build workaround -->
                     <argLine>@{argLine} -Xms1024m -Xmx2048m</argLine>
                     <argLine>-Dfile.encoding=UTF-8</argLine>
+                    <argLine>--add-opens java.base/java.lang=ALL_UNNAMED</argLine>
```

Comment:
Actually it's not recommended to add `--add-opens` here. It's not a common solution.